src/scriptworker/log.py

```diff
@@ -57,6 +57,12 @@ def update_logging_config(context: Any, log_name: Optional[str] = None, file_nam
         # If we rotate the log file via logrotate.d, let's watch the file
         # so we can automatically close/reopen on move.
         handler = logging.handlers.WatchedFileHandler(path)  # type: ignore
+    elif context.config.get("log_max_bytes", 0) or context.config.get("log_max_backups", 0):
+        handler = logging.handlers.RotatingFileHandler(  # type: ignore
+            filename=path,
+            maxBytes=context.config.get("log_max_bytes", 0),
+            backupCount=context.config.get("log_max_backups", 0),
```

Comment:
at which point we don't need the `get`s here.

Reply:
Right :)